sample/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "mt-link-javascript-sdk-sample",
+  "version": "0.1.0",
+  "author": "Moneytree",
+  "description": "An example of the Moneytree Web SDK.",
+  "license": "MIT",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "webpack-dev-server --config webpack.config.js"
```

```suggestion
    "start": "webpack-dev-server"
```

Can we remove `--config webpack.config.js` since it's default config place, or is it for explicitly purpose?